.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+<!--
+  ~ Housekeeping Checklist ~
+  Before submitting a pull request, please make sure the following is done.
+  If a task does not apply, write [na] instead of checking the box.
+
+- [ ] run `yarn` in the repository root
+- [ ] if you've fixed a bug or added code that should be tested, add tests
+- [ ] with the UI running, test if your code breaks existing flows with `yarn e2e:run` at `packages/arb-token-bridge-ui`
+
+#### From `packages/arb-token-bridge-ui` and `packages/token-bridge-sdk`
+
+- [ ] lint your code with eslint, i.e. run `yarn lint`
+- [ ] fix your code with eslint, i.e. run `yarn lint:fix`
+- [ ] format your code with `yarn prettier:format`
```

Comment:
Can we automate linting and formatting steps in pre-commit / pre-merge type hooks?

Reply:
last time our conclusion is to not use pre-commit hooks
we are currently checking format on CI so unless there's a clear need for pre-push hook, we won't add one

Reply:
I don't think it should be enforced in local. I'm fine with developers pushing code that would not pass CI for whatever reasons, but CI should catch it and prevent PR being merged